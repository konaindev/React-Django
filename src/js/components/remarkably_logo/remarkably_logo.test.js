import renderer from "react-test-renderer";
import { MemoryRouter } from "react-router-dom";

import RemarkablyLogo from "./index";

describe("RemarkablyLogo", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<MemoryRouter><RemarkablyLogo /></MemoryRouter>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
