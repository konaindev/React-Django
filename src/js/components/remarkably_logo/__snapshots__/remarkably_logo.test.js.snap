// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RemarkablyLogo renders correctly 1`] = `
<div
  className="remarkably-logo"
>
  Â 
</div>
`;
